runtime/major_gc.c

```diff
@@ -554,10 +563,136 @@ static void mark_ephe_aux (struct mark_stack *stk, intnat *work,
   }
 }
 
+
+#define Pb_size 256
```

Comment:
```suggestion
#define Pb_size (1 << 8)
```
To make it clear that `Pb_size` has to be a power of two.